### ISSUE TITLE
Tags do not affect search results

### ISSUE THREAD

**Issue by Shugabuga** [Krystalite/Krystalite-Installer#1]
The tags listed on the app install page are non-functional and don’t affect search results as they should.

**Comment by Banakin** [comment]
Yeah, they are more for aesthetics and all I used was plain framework7 😐
